frontend/src/layout/navigation-3000/components/NavbarButton.tsx

```diff
@@ -130,7 +130,7 @@ export const NavbarButton: FunctionComponent<NavbarButtonProps> = React.forwardR
                         title={<SidebarChangeNoticeContent notices={notices} onAcknowledged={onAcknowledged} />}
                         placement={notices[0].placement ?? 'right'}
                         delayMs={0}
-                        visible={true}
+                        open={true}
```

Comment:
What's the reasoning behind renaming from `visible` to `open`? While modals are "opened" in common speech, I haven't seen people refer to tooltips as "open", I think "visible" or "shown" is more common

Reply:
No strong opinions. I just aligned it to the internal naming on the `useFloating` hook. I've reverted to keep the external api as it was